client/state/guided-tours/selectors/index.js

```diff
@@ -59,7 +60,7 @@ const getToursFromFeaturesReached = createSelector(
  * recently and in the past.
  */
 const getToursSeen = createSelector(
-	( state ) => [ ...new Set( map( getToursHistory( state ), 'tourName' ) ) ],
+	( state ) => [ ...new Set( getToursHistory( state ).map( ( tour ) => tour.tourName ) ) ],
```

Comment:
At first I thought `getToursHistory()` can be `null`, but it's fine since we have a default value set for 'guided-tours-history' in `DEFAULT_PREFERENCE_VALUES`.